.github/scripts/hermetic_library_generation.sh

```diff
@@ -88,6 +78,9 @@ fi
 git show "${target_branch}":"${generation_config}" > "${baseline_generation_config}"
 config_diff=$(diff "${generation_config}" "${baseline_generation_config}" || true)
 
+# parse image tag from the generation configuration.
+image_tag=$(grep "gapic_generator_version" "${generation_config}" | cut -d ':' -f 2 | xargs)
```

Comment:
what's the purpose of `xargs`?

Reply:
remove leading and trailing white space.